src/Engine/Objects/NPC.h

```diff
@@ -23,7 +23,7 @@ bool CheckHiredNPCSpeciality(NPCProf prof);
 int UseNPCSkill(NPCProf profession, int id);
 void NPCHireableDialogPrepare();
 void _4B4224_UpdateNPCTopics(int _this);
-const char *GetProfessionActionText(NPCProf prof);
+const std::string& GetProfessionActionText(NPCProf prof);
```

Comment:
Code style: `const std::string &GetProfessionActionText`

Reply:
lower case g in getProfessionActionText too?